apiserver/facades/agent/storageprovisioner/storageprovisioner.go

```diff
@@ -32,19 +29,13 @@ import (
 	internalerrors "github.com/juju/juju/internal/errors"
 	"github.com/juju/juju/internal/storage"
 	"github.com/juju/juju/rpc/params"
-	"github.com/juju/juju/state"
 )
 
 // StorageProvisionerAPIv4 provides the StorageProvisioner API v4 facade.
 type StorageProvisionerAPIv4 struct {
-	*common.LifeGetter
-	*common.InstanceIdGetter
```

Comment:
There was no need to remove this one. It was full moved over to Dqlite and is still required and implemented end to end.

Reply:
We are wanting to get rid of these facade plugins and re-implement the logic on each facade where possible. We are prepared to forego a bit of DRY to avoid all the issues we've encountered in the past with facade versioning etc.